### ISSUE TITLE
Regression: Mirror's Edge [NPUB30838] Access Violation Reading Location 0x8c

### ISSUE THREAD

**Issue by NTZ1989** [RPCS3/rpcs3#4633]
Hello. I've tested the game tonight on the build 0.0.5-6802 and it worked fine, runs the opening prefectly and I can play it with somes little hangs. From this version of the emulator on, it gives an error on purple "Access Violation Reading Location 0x8c" the game gives just black screen.

I tested it on these builds, it just loads and right after that comes a black screen and the fps' counter freezes:

 0.0.5-6807 same error
 0.0.5-6814 same error
 0.0.5-6815 same error

CPU Ryzen 1700 | GTX 980ti | Windows 10 1803.
Thanks by the help and keep doing your great job!
![error mirror s edge](https://user-images.githubusercontent.com/39391129/40336758-0381ad6c-5d42-11e8-825a-2b9ff955fd83.png)
[RPC3 log version  0.0.5-6815 .txt](https://github.com/RPCS3/rpcs3/files/2024524/RPC3.log.version.0.0.5-6815.txt)

**Comment by Parthorisian** [comment]
You must upload(following rules) compressed only gz-files(zip, 7z, rar or another archives) like this way [RPC3_mirrorEdge.zip](https://github.com/RPCS3/rpcs3/files/2025783/RPC3_mirrorEdge.zip)

**Comment by AniLeo** [comment]
Test master, it's possible this was a regression with savedata and already fixed on master

**Comment by AniLeo** [comment]
Then this regression reported is fixed, game is back at the status it was.

**Comment by NTZ1989** [comment]
Yeah! Even though it hangs eventually in the game after 5-10 minutes of gameplay, the performance was improved just need to fix this issue then it'll be completely playable. May I or you close this report?

**Comment by NTZ1989** [comment]
@AniLeo Hello!
I've tested build 6841 and it fixed the issue of the emulator hangs after to beat the trainning mode, it loads the story mode correctly, I could play the game until the third mission without fails. First I tried to use the standard settings it hanged the emulator just in the end of the first mission, I tested somes custom settings to attempt sharp the performance, I dismarked all "Additional Settings" and it worked without any fails just somes little hangs that starts after from first mission on. Here I took a print of my settings, maybe it can help someone.

![custom seetings mirror s edge 1](https://user-images.githubusercontent.com/39391129/40443169-d8438160-5e9b-11e8-9925-81072cd68b08.png)
![custom seetings mirror s edge 2](https://user-images.githubusercontent.com/39391129/40443170-d87073c8-5e9b-11e8-8960-3ccaf5b191e6.png)

The performance is fine, 30 fps most of the time, somes little hangs mainly after the first mission. Note this doesn't affect the gameplay or the playability of the game but somes NPCs *in game*, when they speak it plays mute, just Faith and the Narrator plays correctly. During the Cutscene the audio plays perfectly, no mute NPCs are noticed, this just occurs during in game.

Thanks very much, Your are great team!
